snow/engine/snowman/bootstrap/bootstrapper.go

```diff
@@ -471,15 +458,12 @@ func (b *Bootstrapper) Ancestors(ctx context.Context, nodeID ids.NodeID, request
 			zap.Uint32("requestID", requestID),
 		)
 
-		b.markUnavailable(nodeID)
+		b.PeerTracker.RegisterFailure(nodeID)
```

Comment:
This marks a node as unresponsive even though it actually responded... feels like we should be using `RegisterResponse` with a bandwidth of `0` instead?

Reply:
I don't think we want to sample this person again - so we should mark them as unresponsive.

Reply:
By returning no blocks here, the peer is saying that they don't have the requested block history, which can currently happen for state synced C-chain nodes.